src/__tests_/App.test.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { render, fireEvent, cleanup } from 'test-utils';
+import '@testing-library/jest-dom';
+import App from '../App';
+
+afterEach(cleanup);
+
+test('render App', () => {
+  render(<App />);
+});
```

Comment:
nit: wrap all tests in relevant `describe` blocks

```
describe('App', () => {
  afterEach(cleanup);
  
  test('render App', () => {
    render(<App />);
  });
});
```

describes allow for effective scoping within test suites

Reply:
all current comment changes added in commit 17f4d4f5850e37a51176f3566e276723b217dc34